sbt_common/storage/src/test/scala/uk/ac/wellcome/storage/test/fixtures/LocalVersionedHybridStore.scala

```diff
@@ -62,12 +126,25 @@ trait LocalVersionedHybridStore
     )
 
   def getJsonFor[T <: Id](bucket: Bucket, table: Table, record: T) = {
-    val hybridRecord = Scanamo
-      .get[HybridRecord](dynamoDbClient)(table.name)('id -> record.id)
-      .get
-      .right
-      .get
+    val hybridRecord = getHybridRecord(bucket, table, record.id)
 
     getJsonFromS3(bucket, hybridRecord.s3key).noSpaces
   }
+
+  def getContentFor(bucket: Bucket, table: Table, id: String) = {
+    val hybridRecord = getHybridRecord(bucket, table, id)
+
+    getContentFromS3(bucket, hybridRecord.s3key)
+  }
+
+  private def getHybridRecord(bucket: Bucket, table: Table, id: String) =
+    Scanamo.get[HybridRecord](dynamoDbClient)(table.name)('id -> id) match {
+      case None => throw new RuntimeException(s"No object with id $id found!")
```

Comment:
I'd rather leave it as it was with the .gets. This hides the original error thrown by Scanamo and I think in tests you want to see it

Reply:
I found:
```
Scanamo
      .get[HybridRecord](dynamoDbClient)(table.name)('id -> record.id)
      .get
      .right
      .get
```
To hide the location of the error quite effectively before this change. The error is surfaced in the test output via: `RuntimeException(s"Error reading from dynamo: $error")`

Reply:
Ah I see